tools/cachemgr.cc

```diff
@@ -1086,20 +1086,29 @@ static void
 make_pub_auth(cachemgr_request * req)
 {
     static char buf[1024];
+    char *rfc1738_username = NULL, *rfc1738_passwd = NULL;
+
     safe_free(req->pub_auth);
     debug("cmgr: encoding for pub...\n");
 
     if (!req->passwd || !strlen(req->passwd))
         return;
 
+    /*rfc1738_escape is not reentrant */
```

Comment:
rfc1738_escape() is not reentrant indeed, but "reentrant" is not really the right technical term for describing this bug concern (even a perfectly reentrant function may still be unusuable in this debugs() context), and the comment itself just raises more questions IMO. Please remove that comment.

```suggestion
```